### PR TITLE
Feature/select report bugs

### PR DESCRIPTION
**Pull request by JB12298**

### CODE REVIEW

File: src/EA.Weee.Web/Areas/AATFReturn/ViewModels/SelectReportOptionsViewModel.cs [hunk 1]
```diff
@@ -6,7 +6,10 @@
     using EA.Prsd.Core;
     using EA.Weee.Core.AatfReturn;
     using EA.Weee.Core.DataReturns;
+    using FluentValidation.Attributes;
+    using Validation;
 
+    [Validator(typeof(SelectReportOptionsViewModelValidator))]
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/AATFReturn/ViewModels/SelectReportOptionsViewModel.cs]: Can a unit test be added to ensure the model has this attribute with the SelectReportOptionsViewModelValidator

Comment (JB12298) [reply src/EA.Weee.Web/Areas/AATFReturn/ViewModels/SelectReportOptionsViewModel.cs]: Added

**Comment by JohnPadgettCivica** [comment]
Can you just merge from develop :)

**Comment by JohnPadgettCivica** [comment]
Think just need to merge from develop

**Pull request merged**